src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/prison/PrisonerHdcStatus.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.prison
+
+data class PrisonerHdcStatus(
```

Comment:
HDC endpoint does not always return the expected properties, so I've been cautious with the nullables here.